package.json

```diff
@@ -42,6 +42,7 @@
     "lint:format": "prettier --write 'src/**/*.js' 'app/src/**/*.js'",
     "lint": "eslint . --ext .ts",
     "list-outdated-deps": "npm out; cd app && npm out; true",
+    "prepare": "cd ./app && npm install && cd .. && npm run clean && tsc --build . app && npm run build-app && npm run build-app-static",
```

Comment:
```suggestion
    "prepare": "npm run dev-up && npm run build",
```

Reply:
I will try, but npm install is actually run automatically when you install it as a dependency. I am not sure if it is wise to interfere with it as npm run dev-up calls it as well, but npm run build should be safe I think.

Reply:
I changed it to `cd ./app && npm install && cd .. && npm run build`. When I go full npm, it is going into some kind of cycle. I think, when you call npm install, it will call prepare, and then prepare will call install etc.

Reply:
I'd rather keep the separate tasks we have, then, your `prepare` task is confusing, as it only npm installs the **app**. The current tasks are find and documented in https://github.com/jiahaog/nativefier/blob/master/docs/development.md :

-  `dev-up` for initial npm installs of cli and app
- then `build` to build

Reply:
Oh okay, I'm reading the updated PR description where you say `Fix: package.json completely missings prepare (or even prepublish) which breaks using as git dependency`. Alright.

Reply:
@ronjouch This is not my task at all. This a task required by npm to able to use dependency from git: https://docs.npmjs.com/misc/scripts

If you specify `main` and `bin` in your package.json, you need to have those files there after npm itself calls these on the dependency:
```
npm install
npm run prepare
```
This is also why I cannot pot npm install in the prepare, as npm is also calling it will run into infinite cycle.

Reply:
@exander77 got it, see my comment above:

> Oh okay, I'm reading the updated PR description where you say `Fix: package.json completely missings prepare (or even prepublish) which breaks using as git dependency`. Alright.

Reply:
Yes, I am updating it with new info. As I understand it, when you add a dependency from git, npm calls this on it:
```
npm install
npm run prepare
```
And then checks if main and all bin are there and fails completely if they are not there. :(